.github/workflows/utils/log_parser.py

```diff
@@ -229,4 +230,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    exit(1 if main() else 0)
```

Comment:
could return n_errors instead of 1

Reply:
Usually every return code means something. That's why I changed it so it can be `1`, there are errors, or `0`, everything okay.